idp-core/src/main/java/io/mosip/idp/core/dto/KeyBindingResult.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.idp.core.dto;
+
+import lombok.Data;
+
+@Data
+public class KeyBindingResult {
+
+    private String certificate;
+    private String partnerSpecificToken;
```

Comment:
should this not be partnerSpecificUserToken ?

Reply:
Done, renamed to partnerSpecificUserToken